var/spack/repos/builtin/packages/rocsolver/package.py

```diff
@@ -48,7 +50,7 @@ class Rocsolver(CMakePackage):
                 '4.2.0', '4.3.0', '4.3.1']:
         depends_on('hip@' + ver, type='build', when='@' + ver)
```

Comment:
This is probably a link dep too, cause if you use hip as a compiler you link to its libs too most of the time.

Reply:
Yes, we link to `libamdhip64.so`.